package.json

```diff
@@ -124,8 +126,9 @@
     "@storybook/manager-webpack5": "6.5.15",
     "@storybook/react": "6.5.15",
     "@storybook/theming": "6.5.15",
-    "@swc/core": "1.3.30",
+    "@swc/core": "1.3.37",
     "@swc/helpers": "0.4.14",
+    "@swc-node/register": "1.6.1",
```

Comment:
we cannot bump to 1.6.2 as it will nuke in memory transpilation as it uses under the hood magic that makes it impossible to provide path of which tsconfig should be processed. with that nx register  nor direct swc-node/register won't work. 🫡

https://github.com/swc-project/swc-node/pull/694/files#diff-ee889b20e67f97e369cf078afdaa61291584181ad69c5f4be1da106b017923d8R102